tests/integration/test_utils.py

```diff
@@ -70,6 +70,54 @@ def test_directory_acl():
     assert not os.path.isdir(str(d))
 
 
+def test_user_ownership_int():
```

Comment:
These could be moved to unit tests.

Reply:
This is what I wanna talk about. In my world, unit tests don't change anything -- they shouldn't create any files or change anything.

Reply:
So I'm not familiar with the wider context here, but I personally see no problem with creating temporary files during unit tests. What I'm missing here is a cleanup, though :)

Reply:
You freaked me out! Here's the cleanup: https://github.com/fedora-modularity/conu/pull/64/files#diff-fe8f58742ef88776b0c50edcb8d062bfR84

```
filesystem.py              104 INFO     initializing Directory(path=/tmp/qeruikoznj)
filesystem.py              115 INFO     initialized
filesystem.py               94 INFO     brace yourselves, removing '/tmp/qeruikoznj'
```

Reply:
moved to unit tests

Reply:
Ah, ok, didn't realize that the cleanup would be in the `__exit__` method.